.github/workflows/build-ci.yml

```diff
@@ -37,35 +40,43 @@ jobs:
                     sleep 1
                     done
                     sudo docker exec --tty mongodb $MONGOSH_BIN 127.0.0.1:27017 --eval "rs.initiate({\"_id\":\"rs\",\"members\":[{\"_id\":0,\"host\":\"127.0.0.1:27017\" }]})"
-            -   name: Show MongoDB server status
+
+            -   name: "Show MongoDB server status"
                 run: |
                     if [ "${{ matrix.mongodb }}" = "4.4" ]; then MONGOSH_BIN="mongo"; else MONGOSH_BIN="mongosh"; fi
                     docker exec --tty mongodb $MONGOSH_BIN 127.0.0.1:27017 --eval "db.runCommand({ serverStatus: 1 })"
+
             -   name: "Installing php"
-                uses: shivammathur/setup-php@v2
+                uses: "shivammathur/setup-php@v2"
                 with:
                     php-version: ${{ matrix.php }}
-                    extensions: curl,mbstring,xdebug
-                    coverage: xdebug
-                    tools: composer
-            -   name: Show PHP version
-                run: php -v && composer -V
-            -   name: Show Docker version
+                    extensions: "curl,mbstring,xdebug"
```

Comment:
Not related to the changes, as those are CS changes.

From the action-runners docs/[readme](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md)
> Both Xdebug and PCOV extensions are installed, but only Xdebug is enabled.

Does this get overridden with setting the extensions manually?